src/main/scss/styles.scss

```diff
@@ -20,15 +18,15 @@ html, body {
 }
 
 header {
-	border-bottom: 1px solid #C3C3C3;
+	border-bottom: 1px solid $gray-border;
```

Comment:
Up to you, feel free to do the following in _variables.scss:
```
$color-accent: <the grey color>;

$header-border-color: $color-accent;
$footer-border-color: $color-accent;
```
And modify the variable usage in this file accordingly.